docs/content/Configuration/Multitenancy-Setup.md

```diff
@@ -20,6 +20,13 @@ The options are:
 - `preAggregationsSchema`
 - `queryTransformer`
 
+<!-- prettier-ignore-start -->
+[[warning |]]
+| In the [scheduled refresh](caching#pre-aggregations-background-refresh) leaving [`scheduledRefreshContexts`](config#options-reference-scheduled-refresh-contexts)  
+| unconfigured will lead to issues where the security context will be `undefined`. This is because there is no way for
+| Cube.js to know how to generate a context without the required input.
+<!-- prettier-ignore-end -->
+
```

Comment:
Not sure about the placement of this - I think we should create a section in this (Multitenancy) page specifically for scheduled refreshes, WDYT @keydunov @ovr?

Reply:
We have the issue: https://github.com/cube-js/cube.js/issues/1904
Where users don't know about that they need to use `scheduledRefreshContexts` with multitenancy. It's why I added a reference as warning block to indicate that it's important.

Reply:
@ovr I totally agree we should keep the warning 😄 I just meant maybe we should also make a 'Scheduled Refresh' section on this page and then move the warning there. The current placement of the warning seems a bit disconnected from the rest of the text around it.